tests/basic/pool_function_test.go

```diff
@@ -994,3 +995,107 @@ var _ = Describe("{PoolExpandTestLimits}", func() {
 		dash.VerifyFatal(err != nil, true, "DMThin pool expansion to 20 TB should result in error")
 	})
 })
+
+var _ = Describe("{CheckPoolLabelsAfterAddDisk}", func() {
+
+	var testrailID = 34542906
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542906
+
+	BeforeEach(func() {
+		StartTorpedoTest("CheckPoolLabelsAfterAddDisk",
+			"Initiate pool expansion and Newly set pool labels should persist post pool expand add-disk operation", nil, testrailID)
+		contexts = scheduleApps()
```

Comment:
After this command, Lets validate that scheduled Apps come up successfully using `ValidateApplications` API.
https://github.com/portworx/torpedo/blob/master/tests/common.go#L1891

Reply:
Is that necessary? We are not doing that anywhere else.